### DISCUSSION TITLE
Projection Mapping in Cables gl ?

### DISCUSSION THREAD

**Discussion by felipeinfantino** [cables-gl/cables_docs#303] [Feature Requests] [answered]
Hi, first of all, thanks to the cables.gl team, awesome tool. 

I come from TouchDesigner and wanted to know if there is a projection mapping tool in cables, something equivalent to
https://github.com/Richard-Burns/Warpa

If not, maybe something similar can be implemented with https://github.com/glowbox/maptasticjs (warpa tool is of course better because you can provide a .fbx/.obj and creates all the points for you and lets you warp them individually.)

How feasible would be to implement this ? Can be done through the Cables API ? 

Greetings,

**Comment by pandrr** [comment, marked as answer]
https://cables.gl/op/Ops.Gl.Meshes.QuadWarpTexture you can use this for basic warping. 

i would recommend an external software for more advanced stuff...

maptasticjs: interesting thingie
